tests/integration_test/services_test.py

```
@@ -6904,6 +6904,312 @@ def test_catalog_sync_preprocess_exception(mock_embedding, mock_preprocess, mock
+        url=f"/api/bot/{pytest.bot}/data/integrations/add?sync_type=push_menu",
+        json = payload,
+        headers={"Authorization": pytest.token_type + " " + pytest.access_token}
+    )
+    actual = response.json()
+    assert actual["message"] == "POS Integration Complete"
+    assert actual["error_code"] == 0
+    assert actual["success"]
+    assert "integration/petpooja/push_menu" in actual["data"]
+    assert str(pytest.bot) in actual["data"]
+    sync_url = actual["data"]
+    token = sync_url.split(str(pytest.bot) + "/")[1]
```

Ruff (0.11.9)

Local variable `token` is assigned to but never used

Remove assignment to unused variable `token`

(F841)